auth-agents-common/src/main/java/org/apache/atlas/plugin/conditionevaluator/AtlanDQAllowExceptionCondition.java

```diff
@@ -0,0 +1,105 @@
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.*;
+
+
+public class AtlanDQAllowExceptionCondition extends RangerAbstractConditionEvaluator {
+    private static final Log LOG = LogFactory.getLog(AtlanHasAnyRole.class);
+    protected Set<String> excludedActions = new HashSet<>();
+    public static final String RESOURCE_ENTITY_TYPE                   = "entity-type";
+    public static final String RESOURCE_ENTITY_CLASSIFICATION         = "entity-classification";
+    public static final String RESOURCE_CLASSIFICATION                = "classification";
+    public static final String RESOURCE_ENTITY_ID                     = "entity";
+    public static final String RESOURCE_ENTITY_LABEL                  = "entity-label";
+    public static final String RESOURCE_ENTITY_BUSINESS_METADATA      = "entity-business-metadata";
+    public static final String RESOURCE_ENTITY_OWNER                  = "owner";
+    public static final String RESOURCE_RELATIONSHIP_TYPE             = "relationship-type";
+    public static final String RESOURCE_END_ONE_ENTITY_TYPE           = "end-one-entity-type";
+    public static final String RESOURCE_END_ONE_ENTITY_CLASSIFICATION = "end-one-entity-classification";
+    public static final String RESOURCE_END_ONE_ENTITY_ID             = "end-one-entity";
+    public static final String RESOURCE_END_TWO_ENTITY_TYPE           =  "end-two-entity-type";
+
+
+    @Override
+    public void init() {
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("==> AtlanDQAllowExceptionCondition.init(" + condition + ")");
+        }
+
+        super.init();
+
+        if (condition != null ) {
+            for (String value : condition.getValues()) {
+                excludedActions.add(value.trim());
+            }
+        }
+
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("<== AtlanDQAllowExceptionCondition.init(" + condition + ")");
+        }
+    }
+
+    @Override
+    public boolean isMatched(RangerAccessRequest request) {
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("==> AtlanDQAllowExceptionCondition.isMatched(" + condition + ")");
+        }
+
+        boolean ret = true;
+        RangerAccessRequest	readOnlyRequest = request.getReadOnlyCopy();
+
+        Object[] entityTypeObjects = {
+                readOnlyRequest.getResource().getValue(RESOURCE_ENTITY_TYPE),
+                readOnlyRequest.getResource().getValue(RESOURCE_END_ONE_ENTITY_TYPE),
+                readOnlyRequest.getResource().getValue(RESOURCE_END_TWO_ENTITY_TYPE)
+        };
+
+        List<?> entityTypes = Arrays.stream(entityTypeObjects)
+                .filter(Objects::nonNull)
+                .map(this::convertToList)
+                .findFirst()
+                .orElseGet(ArrayList::new);
+
+        if (entityTypes.isEmpty() || !entityTypes.contains("alpha_DQRule")) {
```

Comment:
instead of AtlanDQAllowExceptionCondition, I suggest writing a logic to exclude the specified asset types.
E.g.
instead of 
``` 
 "policyConditions": [
        {
          "policyConditionType": "excludeDQRules",
          "policyConditionValues": [
            "entity-read"
          ]
        }
      ]
```

You will write it as 
```
      "policyConditions": [
        {
          "policyConditionType": "excludeAssetTypes",
          "policyConditionValues": [
            "alpha_DQRule",
            "any new type we want to exclude later"
          ]
        }
      ]
```

This way this implementation becomes simpler & generic to extend future use cases